app/models/google_calendar/event_wrapper/event.rb

```diff
@@ -25,6 +25,20 @@ def valid?
         start_time.date_time.present? && end_time.date_time.present?
       end
 
+      def all_day?
+        start.date.present?
+      end
+
+      def in_progress?
+        current_time >= start_time.date_time && current_time <= end_time.date_time
+      end
+
+      def finish
```

Comment:
But then you'll make Prima Donna code smell ;) https://github.com/troessner/reek/blob/master/docs/Prima-Donna-Method.md